### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by tomharrigan**
View the rendered readme at https://github.com/alleyinteractive/voicewp/tree/doc-updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,44 +1,375 @@
...
+Create Alexa Skills through WordPress
+
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/voicewp/Lobby)
+
 VoiceWP is a WordPress plugin that integrates with Amazon Alexa to create and enable the creation of Alexa skills.
 
-It allows you to use WordPress as an endpoint.
+## How it Works
+
+VoiceWP creates REST endpoints using the WordPress REST API. These endpoints handle all of the logic of your Alexa skills. When a user interacts with your skill, it sends a Request to an endpoint, which is processed by this plugin, and sends the Response back to the user.
```
Comment (pattiereaves) [inline README.md:11]: should `Request` and `Response` be capitalized?

File: README.md [hunk 2]
```diff
@@ -1,44 +1,375 @@
...
 
-- The minimum required PHP version for VoiceWP is PHP 5.3.
+- Install and activate [Fieldmanager](https://github.com/alleyinteractive/wordpress-fieldmanager/archive/1.0.0.zip).
+- Download the .zip file of this repo and upload to your WordPress site by going to WP Admin and navigating to **Plugins -> Add New**. Select the 'Upload Plugin' button near the top of the top of the screen to upload the .zip file.
+
+## Features
 
-- The minimum required WordPress version for VoiceWP is WordPress Version 4.4.
+- 3 different types of skills can be created out of the box
+	- __Flash Briefings__: Deliver original content to users as part of their flash briefing. For more on Flash Briefings, [see here](https://developer.amazon.com/alexa-skills-kit/flash-briefing).
```
Comment (pattiereaves) [inline README.md:32]: Should flash briefings be capitalized in the second instance?

File: README.md [hunk 3]
```diff
@@ -1,44 +1,375 @@
...
+	- __News__: Allow users to listen to your content/posts
+		- Fully editable 'Welcome', 'Help', and 'Stop' messaging
+		- Presents users with a list of posts to select from
```
Comment (pattiereaves) [inline README.md]: from which to select - shouldn't end on a preposition

File: README.md [hunk 4]
```diff
@@ -1,44 +1,375 @@
 # VoiceWP
 
+Create Alexa Skills through WordPress
+
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/voicewp/Lobby)
```
Comment (pattiereaves) [inline README.md:5]: 💅 fancy!

File: README.md [hunk 5]
```diff
@@ -1,44 +1,375 @@
...
+- 3 different types of skills can be created out of the box
+	- __Flash Briefings__: Deliver original content to users as part of their flash briefing. For more on Flash Briefings, [see here](https://developer.amazon.com/alexa-skills-kit/flash-briefing).
+	- __News__: Allow users to listen to your content/posts
+		- Fully editable 'Welcome', 'Help', and 'Stop' messaging
+		- Presents users with a list of posts to select from
+		- Selected content/post is read in full
+		- Users can ask for content from a specific category/taxonomy
+		- Full SSML support within post content
+		- Full utterances/intent schema provided as starting point to easily copy/paste into theh Amazon Developer console
+		- Does your site use acronyms or other abbreviations? Settings screen provides an interface for defining items Alexa should pronounce a certain way (For example, a site may want all instances of 'DoT' within text to be read as 'Department of transportation' )
```
Comment (pattiereaves) [inline README.md]: Transportation should be capitalized

File: README.md [hunk 6]
```diff
@@ -1,44 +1,375 @@
...
+	- __Facts/Quotes__: Create simple skills for serving facts or quotes on your favorite topics. For example, 'Cat Facts', or 'Developer Quotes'.
+		- Ability to include custom images as part of the app cards.
+		- Ability to include this as a subset of functionality within the main News skill.
+- Developers can create completely new types of skills or customize the existing functionality of the plugin via provided hooks, filters and functions. Documentation [outlined here](#documentation).
 
-- The [Fieldmanager](http://fieldmanager.org) plugin is required for VoiceWP to function properly. It is used for custom field, meta, and settings screens.
+## Contribute
 
-- An SSL certificate is required on your site. You can use [Let's Encrypt](https://letsencrypt.org/).
+All development of VoiceWP happens transparently on Github. [Github Issues](https://github.com/alleyinteractive/voicewp/issues) are used for identifying and discussing bugs and features. Code contributions, whether fixes or enhancements, should be submitted as Pull Requests.
```
Comment (pattiereaves) [inline README.md]: pull requests should be lowercase

File: README.md [hunk 7]
```diff
@@ -1,44 +1,375 @@
...
-- The minimum required WordPress version for VoiceWP is WordPress Version 4.4.
+- 3 different types of skills can be created out of the box
+	- __Flash Briefings__: Deliver original content to users as part of their flash briefing. For more on Flash Briefings, [see here](https://developer.amazon.com/alexa-skills-kit/flash-briefing).
+	- __News__: Allow users to listen to your content/posts
+		- Fully editable 'Welcome', 'Help', and 'Stop' messaging
+		- Presents users with a list of posts to select from
+		- Selected content/post is read in full
+		- Users can ask for content from a specific category/taxonomy
+		- Full SSML support within post content
+		- Full utterances/intent schema provided as starting point to easily copy/paste into theh Amazon Developer console
```
Comment (pattiereaves) [inline README.md]: typo - theh

File: README.md [hunk 8]
```diff
@@ -1,44 +1,375 @@
...
+
+The main Alexa Skill Settings screen is located at __Tools -> Alexa Skill Settings__
+
+![setting_menu](https://user-images.githubusercontent.com/784167/27144151-c7ff3b2e-50fe-11e7-8043-431090ec8646.png)
+
+The most important field that needs to be filled in for the news functionality to work is the `News skill ID` field. The Skill ID can be retrieved from the Amazon Developer Console:
+
+![skill_id](https://user-images.githubusercontent.com/784167/27146948-11a683cc-5109-11e7-9afc-502b85696cfa.png)
+
+The top of the settings screen provides the ability to set and customize a number of messages used in the skill. The Welcome message is triggered when a user opens your skill, for example 'Alexa, open Developer Quotes'. (This occurs by Alexa sending a 'LanchIntent' to your skill). The welcome message can prompt the user to take a certain action, or give them a brief intro to the skill.
```
Comment (pattiereaves) [inline README.md]: Welcome is capitalized in the first instance and lowercase in the subsequent ones.

File: README.md [hunk 9]
```diff
@@ -1,44 +1,375 @@
...
+
+The Help message should provide information for the different actions a user can take and occurs when the user asks for help. This occurs by Alexa sending a 'AMAZON.HelpIntent' to your skill.
+e
+The Stop message is completely optional. When a user asks the skill to stop or cancel, you can provide a parting message.
+
+![intent_messages](https://user-images.githubusercontent.com/784167/27147008-496cb858-5109-11e7-8565-91d288aab637.png)
+
+Many sites have a set of abreviations, acronyms, etc. That should be spoken differently by Alexa than how they appear in text. For example, the state abbreviation 'TN' should be pronounced as 'Tennessee'. A meta box with repeating field is available for you to define these words and phrases. Simply add the item in the first field, and how it should be read by Alexa in the second field.
```
Comment (pattiereaves) [inline README.md]: abreviations -> abbreviations

**Review by pattiereaves — APPROVED** [review]
🎙

File: README.md (hunk 1 above)
Comment (tomharrigan) [reply README.md:11]: the plugin immediately turns them into objects of those classes, so to keep the consistency of the class names i went with capitalized

**Pull request merged**